From: Johannes Thumshirn <morbidrsa@googlemail.com>
Date: Sat, 5 May 2012 16:31:50 +0200
Subject: [026/106] staging: line6/midibuf.c changed printk(KERN_DEBUG, ... to
 pr_debug(

commit 8878451510b11ee9095198f42bf1d59e6bab66fb upstream.

Changed printk(KERN_DEBUG, ...) call to pr_debug call in function
void line6_midibuf_status(struct MidiBuffer *this)

Signed-off-by: Johannes Thumshirn <morbidrsa@googlemail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/line6/midibuf.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: linux-3.2.46/drivers/staging/line6/midibuf.c
===================================================================
--- linux-3.2.46.orig/drivers/staging/line6/midibuf.c	2013-07-26 18:52:54.000000000 +0000
+++ linux-3.2.46/drivers/staging/line6/midibuf.c	2013-07-26 19:28:32.000000000 +0000
@@ -64,7 +64,7 @@
 
 void line6_midibuf_status(struct MidiBuffer *this)
 {
-	printk(KERN_DEBUG "midibuf size=%d split=%d pos_read=%d pos_write=%d "
+	pr_debug("midibuf size=%d split=%d pos_read=%d pos_write=%d "
 	       "full=%d command_prev=%02x\n", this->size, this->split,
 	       this->pos_read, this->pos_write, this->full, this->command_prev);
 }
